assets/js/search.js

```diff
@@ -20,17 +21,39 @@ $(function() {
 
   // Submit form on Sort change event.
   $( '#select_order' ).off( 'change' ).on( 'change', function() {
+    var $es = $('#es', $search_form);
+
+	if ( $es.length > 0 ) {
+	  is_elastic_search = true;
+    }
     $( '#orderby', $search_form ).val( $( this ).val() ).parent().submit();
     return false;
   });
 
   // Submit form on Filter click event or on Apply button click event.
   $( 'input[name^="f["]:not(.modal-checkbox), .applybtn' ).off( 'click' ).on( 'click', function() {
+    var $es = $('#es', $search_form);
```

Comment:
This block of code (determining if it is elastic search ) appears sometimes, maybe it could be extracted in a function.

Reply:
Applied suggestions